eng/pipelines/templates/steps/common.yml

```diff
@@ -6,9 +6,9 @@ steps:
     inputs:
       versionSpec: "3.6"
 
-  - template: eng/pipelines/templates/scripts/verify-agent-os.yml@azure-sdk-tools
+  - template: /eng/common/pipelines/templates/steps/verify-agent-os.yml
```

Comment:
As is this looks correct but please wait until we finish with the feedback in https://github.com/Azure/azure-sdk-tools/pull/610#discussion_r424061807 to see if there might be changes.